src/formpack/reporting/export.py

```diff
@@ -680,13 +668,9 @@ def to_geojson(
                         # etc. so we skip over those too. Also skip over those
                         # fields that are blank or are included regardless of
                         # the specified filtered fields.
-                        if not all(
-                            re.search(fr'_?{geo}_?', label) is None
-                            for geo in all_geo_field_names
-                        ) or (
-                            len(str(row_value)) == 0
-                            or label in ('_id', '_index')
-                        ):
+                        if label in all_geo_field_names:
+                            continue
+                        if not row_value or label in ('_id', '_index'):
```

Comment:
if `_index` is the the reference used for cross-referencing responses to repeating groups, then i'd agree it's useless to include here. i'm not sure about `_id`, though. (can people remove these with the forthcoming custom export feature so we don't have to hard-code them here?)

Reply:
ok, no:
> Also skip over those fields that are blank or are included regardless of the specified filtered fields

note to self: check if these are always included (perhaps this code simply prevents them from being included twice)

Reply:
no, they are not included at all based on the unit tests
:eyes:

Reply:
Correct... I think I needed that when I hadn't integrated filtering into the rest of the exports, but now that it is, it works as expected 🙌

Reply:
Perfect!